maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/SwipeRouteHandler.swift

```diff
@@ -0,0 +1,65 @@
+import FlyingFox
+import XCTest
+import os
+
+class SwipeRouteHandler: RouteHandler {
+    
+    private let logger = Logger(subsystem: Bundle.main.bundleIdentifier!, category: "SwipeRouteHandler")
+    
+    func handle(request: FlyingFox.HTTPRequest) async throws -> FlyingFox.HTTPResponse {
+        guard let appId = request.query["appId"] else {
+            logger.error("Requested view hierarchy for an invalid appId")
+            return HTTPResponse(statusCode: HTTPStatusCode.badRequest)
+        }
+        
+        let decoder = JSONDecoder()
+        
+        guard let requestBody = try? decoder.decode(SwipeRequest.self, from: request.body) else {
+            let errorData = handleError(message: "incorrect request body provided")
+            return HTTPResponse(statusCode: HTTPStatusCode.badRequest, body: errorData)
+        }
+        
+        let response = await MainActor.run {
+            let xcuiApplication = XCUIApplication(bundleIdentifier: appId)
+            
+            let element = xcuiApplication
+            
+            let velocity: XCUIGestureVelocity
+            if let v = requestBody.velocity {
+                velocity = XCUIGestureVelocity(CGFloat(v))
+            } else {
+                velocity = XCUIGestureVelocity.default
+            }
+            
+            let startPoint = element.coordinate(withNormalizedOffset: CGVector(
+                    dx: CGFloat(requestBody.startX),
+                    dy: CGFloat(requestBody.startY)
+            ))
+            
+            let endPoint = element.coordinate(withNormalizedOffset: CGVector(
+                    dx: CGFloat(requestBody.endX),
+                    dy: CGFloat(requestBody.endY)
+            ))
+            
+            startPoint.press(
```

Comment:
Let's log here the incoming request start, end, and velocity with the logger that are resolved for debugging.